internal/api/export/batcher.go

```diff
@@ -98,7 +98,7 @@ func (s *Server) maybeCreateBatches(ctx context.Context, ec *model.ExportConfig,
 		return 0, fmt.Errorf("fetching most recent batch for config %d: %w", ec.ConfigID, err)
 	}
 
-	ranges := makeBatchRanges(ec.Period, latestEnd, now)
+	ranges := makeBatchRanges(ec.Period, latestEnd, now, s.config.TruncateWindow)
```

Comment:
I have an informal, unwritten, totally-from-my-experience-of-running-go-apps-at-scale rule that once function arity reaches 4 inputs, it should either be restructured into smaller methods or accept an input/request struct:

```go
type MakeBatchRangesInput struct {
  Period string,
  # ...
}

func makeBatchRanges(i *MakeBatchRangesInput) {}
```

Reply:
Filed https://github.com/google/exposure-notifications-server/issues/312